### DISCUSSION TITLE
Version 4 release - how to know when it is ready

### DISCUSSION THREAD

**Discussion by gbjbasdw** [GerryDazoo/Slinger#504] [Q&A]
Am I right in thinking that I will see a "V4.0_ReleaseNotes.pdf" appear in the left hand side of the main Slinger Github page when it is released ?

I've just been reading the V4 notes that were linked in one of the threads. Slingerplayer sounds just what we needed as it can form part of the Slinger eco system rather than having to use an externally provided player.

The AP stuff lost me I'm afraid but hopefully I wont need it anyway !

I'm off on holiday next Sunday so it will be great to try Slinger out for real. I think I will wait until I get back until I try V4 as I don't want to screw anything up while I am away :-(

As always, thanks a million Gerry.

**Comment by GerryDazoo** [comment]
Yes, I will make an Announcement in these discussions when I've updated github with the latest. You can get a preview over on Discord. slinger-dev thread.

**Reply by socelillyhammer** [reply]
any ideas on someone creating an app? or will that need funding?

**Reply by GerryDazoo** [reply]
What "app" are you referring to?
